packages/flame/test/components/position_component_test.dart

```diff
@@ -465,6 +465,40 @@ void main() {
       expect(component.center, closeToVector(x, y, epsilon: 1e-14));
     });
 
+    test('isHorizontallyFlipped', () {
+      final component = PositionComponent()
+        ..size = Vector2(10, 10)
+        ..position = Vector2(50, 20)
+        ..anchor = const Anchor(0.6, 0.8);
+
+      expect(component.isFlippedHorizontally, isFalse);
+      component.flipHorizontally();
+      expect(component.isFlippedHorizontally, isTrue);
+      component.flipHorizontally();
+      expect(component.isFlippedHorizontally, isFalse);
+      component.flipHorizontallyAroundCenter();
+      expect(component.isFlippedHorizontally, isTrue);
+      component.flipHorizontallyAroundCenter();
+      expect(component.isFlippedHorizontally, isFalse);
```

Comment:
I wonder what happens if "flip" with one version and "unflip" with the other

Reply:
I think that it will move the component from its original position (which I think makes sense), if I remember correctly this is documented somewhere, maybe on the dartdocs.